server/sonar-server/src/main/java/org/sonar/server/computation/issue/TrackerRawInputFactory.java

```diff
@@ -95,7 +100,9 @@ protected LineHashSequence loadLineHashSequence() {
           ScannerReport.Issue reportIssue = reportIssues.next();
           if (isIssueOnUnsupportedCommonRule(reportIssue)) {
             DefaultIssue issue = toIssue(getLineHashSequence(), reportIssue);
-            result.add(issue);
+            if (issueFilter.accept(issue, component)) {
+              result.add(issue);
+            }
           } else {
             Loggers.get(getClass()).debug("Ignored issue from analysis report on rule {}:{}", reportIssue.getRuleRepository(), reportIssue.getRuleKey());
```

Comment:
The log is telling that the issue has been ignored because of the settings. 
Rule is supported, but ignored